parlai/agents/fid/fid.py

```diff
@@ -352,6 +353,14 @@ def get_retrieved_knowledge(self, message):
 
     def _set_query_vec(self, observation: Message) -> Message:
         retrieved_docs = self.get_retrieved_knowledge(observation)
+        if len(retrieved_docs) > self._n_docs:
```

Comment:
@klshuster picking the right documents, if there are more than `n_docs`, depends a lot on the data set. I add this warning here and trim it down to the first `n_docs`. I hope this properly addresses your comment. LMK.

Reply:
sounds good to me. given how interactive this retriever is (i.e., interactive with the user --> the user has total control over what documents are being passed), it makes sense that the onus is on them to either 1) provide the right number of documents, or 2) set `--n-docs` correctly